### PR TITLE
DP-25197 patch core to avoid scrolling body when showing a modal

### PR DESCRIPTION
**Pull request by beto-aveiga**
**Description:**
Disables body overflow when a modal is shown.

**Jira:** (Skip unless you are MA staff)
DP-25197

**To Test:**
- [ ] Add steps to test this feature

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

**Comment by deviantintegral** [comment]
@beto-aveiga it looks good for the image modal:

image

However, for the Views modal it's both cut off at the top and hides the close button:

image

Did your prior PR work in the views case?

Once you've looked at this, please add testing steps to the PR description and screenshots showing each of the cases in the ticket are working. Thank you!

File: composer.json [hunk 1]
```diff
@@ -322,6 +322,7 @@
                 "Invalid argument supplied for foreach SettingsForm->prepareHeadersDisplay (https://www.drupal.org/project/access_unpublished/issues/3252499)": "https://www.drupal.org/files/issues/2021-12-03/3252499-warning-invalid-argument-2.patch"
             },
             "drupal/core": {
+                "Disable body-level scrolling when a dialog is open as a modal #2707291" : "https://www.drupal.org/files/issues/2022-01-30/2707291-62_0.patch",
```
Comment (deviantintegral) [inline composer.json:325]: FYI we recently adopted a policy at Lullabot of committing patches to the repository: https://architecture.lullabot.com/adr/20220429-composer-patch-files/

It's something to consider! cc @tom-fleming @weitzman .

Comment (weitzman) [reply composer.json:325]: AFAIK, It is 100% safe to reference patches from drupal.org. They cant be changed. Its less safe to reference Gitlab patch URLs as those can change.

**Comment by beto-aveiga** [comment]
@deviantintegral is it okay if I re-open my previous PR and check if it works as expected?

**Comment by deviantintegral** [comment]
@beto-aveiga yes, thank you!

**Pull request closed without merging**